web-app-pix2info-python/src/.gcloudignore

```diff
@@ -0,0 +1,14 @@
+# This file specifies files that are *not* uploaded to Google Cloud
```

Comment:
I never knew this type of file existed!

Reply:
👍 Like `.gitignore` but for deployments.